workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -83,3 +84,19 @@ def write_chunks(config: UserConfig):
     chunks = get_chunks(diagnostic_file_configs)
     with open("chunks.yaml", "w") as f:
         yaml.safe_dump(chunks, f)
+
+
+def write_existing_rundir_items(filename: str = "existing_files.yaml"):
+    """Write list of files which currently exist in rundir. Ignores ./logs.txt.
+    
+    .. warning::
+        Only valid at runtime
+    """
+    items = []
+    for root, dirs, files in os.walk("."):
+        for name in files:
+            items.append(os.path.join(root, name))
+    if "./logs.txt" in items:
+        items.remove("./logs.txt")
```

Comment:
This is a bit hacky... not sure how to handle this beyond defining a new command line function which is called in `runfv3` before the `mpirun RUNFILE | tee logs.txt` call.

Reply:
This point is no longer relevant.